### DISCUSSION TITLE
Minestom exception in ComponentLogger during Minestom#init()

### DISCUSSION THREAD

**Discussion by Tofaa2** [Minestom/Minestom#1702] [Help] [answered]
Here's the error

https://paste.gg/p/anonymous/712fc572211c408b89f443d0303002c0

Stacktrace points at  
```kotlin
val server = MinecraftServer.init()
```

It is the first line in the main method

**Comment by ZakShearman** [comment, marked as answer]
Can you send the full version line in your maven/gradle please

**Reply by mworzala** [reply]
The whole maven/gradle file preferably. This looks like using shadowjar minify badly or something

**Reply by Tofaa2** [reply]
Apparently shadow jar's minimize() breaks Minestom.
